client/src/pages/ProfileDetails/useStyles.ts

```diff
@@ -27,6 +27,19 @@ const useStyles = makeStyles((theme: Theme) => ({
     borderRadius: 10,
     marginRight: 20,
   },
+  reviewsContainer: {
+    display: 'flex',
```

Comment:
These flexbox styles can be added to the Box component directly. 

Personally, for any significant styles I apply them as props to the MUI component, for any other styles I use a CSS class.